smartsim/settings/alpsSettings.py

```diff
@@ -103,6 +112,21 @@ def set_hostlist(self, host_list):
         if not all([isinstance(host, str) for host in host_list]):
             raise TypeError("host_list argument must be list of strings")
         self.run_args["nodelist"] = ",".join(host_list)
+        
+    def set_excludelist(self, host_list):
```

Comment:
``set_excluded_hosts``?

Reply:
I think I got it from Slurm, but I like yours more.